kolibri/core/public/api_urls.py

```diff
@@ -36,9 +39,21 @@
 router.register(r"info", InfoViewSet, base_name="info")
 router.register(r"syncqueue", SyncQueueViewSet, base_name="syncqueue")
 
+public_content_v2_router = routers.SimpleRouter()
+public_content_v2_router.register(
+    r"channel", PublicChannelMetadataViewSet, base_name="publicchannel"
+)
+public_content_v2_router.register(
+    r"contentnode", ContentNodeViewset, base_name="publiccontentnode"
+)
+public_content_v2_router.register(
+    r"contentnode_tree", ContentNodeTreeViewset, base_name="publiccontentnode_tree"
+)
+
 # Add public api endpoints
 urlpatterns = [
     url(r"^", include(router.urls)),
+    url(r"v2/", include(public_content_v2_router.urls)),
```

Comment:
This does not match with the examples in the review guidance of this PR description. I had to change them to
http://127.0.0.1:8000/api/public/v2/channel/?format=json&baseurl=http://127.0.0.1:8080/
http://127.0.0.1:8000/api/public/v2/contentnode/?format=json&max_results=25&baseurl=http://127.0.0.1:8080/
etc.
Please, change the description for future reviewers